### PR TITLE
better kmcoutput every 1e9 steps

### PR DESCRIPTION
**Pull request by JensWehner**
Output of diffusion tensor every 1e9 steps

### CODE REVIEW

File: src/libxtp/calculators/kmcmultiple.cc [hunk 1]
```diff
@@ -284,6 +286,12 @@ void KMCMultiple::RunVSSM(ctp::Topology *top)
                 avgdiffusiontensor += (_carriers[i]->dr_travelled)|(_carriers[i]->dr_travelled);
             }
         }
+        
+        if(step!=0 && step%(int(1e9))==0){
```
Comment (baumeier) [inline src/libxtp/calculators/kmcmultiple.cc]: Why have 1e9 hard coded?

File: src/libxtp/calculators/kmcmultiple.cc [hunk 2]
```diff
@@ -284,6 +286,12 @@ void KMCMultiple::RunVSSM(ctp::Topology *top)
...
+            unsigned long diffusionsteps=step/diffusionresolution;
+            tools::matrix result=avgdiffusiontensor/(diffusionsteps*2*simtime*_numberofcharges);
+            cout<<endl<<"Step: "<<step<<" Diffusion tensor averaged over all carriers (nm^2/s):" << endl << result << endl;
```
Comment (baumeier) [inline src/libxtp/calculators/kmcmultiple.cc]: Can we have the same for average velocity, useful in mobility calculation

File: src/libxtp/calculators/kmcmultiple.cc (hunk 1 above)
Comment (JensWehner) [reply src/libxtp/calculators/kmcmultiple.cc]: yes, we can set it as a default option.

File: src/libxtp/calculators/kmcmultiple.cc (hunk 2 above)
Comment (JensWehner) [reply src/libxtp/calculators/kmcmultiple.cc]: yes, okay

**Pull request merged**